mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/UserScreen.kt

```diff
@@ -29,15 +40,88 @@ fun UserScreen(
     onMatchClick: (ChessMatch) -> Unit,
     lazyColumnState: LazyListState,
     modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(),
 ) {
+  val strings = LocalLocalizedStrings.current
+
   LazyColumn(
       state = lazyColumnState,
       verticalArrangement = Arrangement.Top,
       horizontalAlignment = Alignment.CenterHorizontally,
       modifier = modifier,
+      contentPadding = contentPadding,
   ) {
     item { header() }
     stickyHeader { profileTabBar() }
-    items(matches) { match -> Match(match, { onMatchClick(match) }) }
+    items(matches) { match ->
+      val title = strings.profileMatchTitle(match.adversary)
+      val subtitle = chooseSubtitle(strings, match.matchResult, match.numberOfMoves)
+      Match(title, subtitle, ChessIcons.WhiteKing)
+    }
   }
 }
+
+/**
+ * Composes a Match log using a match [title], [subtitle] and an [icon]
+ * @param title match title
+ * @param subtitle match subtitle info
+ * @param icon match icon
+ * @param modifier the [Modifier] for this composable.
+ */
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun Match(
+    title: String,
+    subtitle: String,
+    icon: Painter,
+    modifier: Modifier = Modifier,
+) {
+  val titles = title.split(' ', limit = 2)
+  val profileAgainst = titles[0]
+  val profileOpponent = titles[1]

```

Comment:
[Question] I don't really understand why is the  `profileAgainst` and `profileOpponent` deduced form the title. Don't they deserve their own arguments ? Plus what it's the difference between them the names are not clear to me.

Reply:
Hey Badr,
Yes indeed I was going to have them have separate arguments in the LocalizedStrings (which is why I also added the profileAgainst string value) then decided it may be best to keep the original title argument and split it in half in order to give each part a different color just like in the Figma mockup, while still utilizing the profileMatchTitle function in LocalizedStrings.

Reply:
I 100% agree with @BadrTad. If the format of the title changes, this will go terribly wrong.

Reply:
Fixed in 87b7aea